Wangscape/tilegen/CoordinatePacker.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+#include <assert.h>
```

Comment:
I removed the assertions I used in this file because failing an assertion outside of `(EXPECT|ASSERT)_DEATH` causes GTest to crash. This `#include` should just be removed. Also, in future code assertions should only be used for cases that cannot occur in normal use or in tests.